web/src/components/itemList.js

```diff
@@ -58,23 +61,34 @@ class ItemList extends Component {
         const cards = [];
         for(let i = 0; i < this.state.items.length; i++){
             cards.push(<Col sm={6}>
-                <Card style={{ marginBottom: '10px' }}>
-                    <Card.Img variant="top" src={this.state.items[i].imgUrl} style={{width: '100%', height: '20vw', objectFit: 'cover'}} />
-                <Card.Body>
-                    <Card.Title>{this.state.items[i].name}</Card.Title>
-                    <Card.Text>
-                        <NumberFormat value={this.state.items[i].price * (this.state.currency[i] ? this.rate : 1)} displayType={'text'} thousandSeparator={true} prefix={this.state.currency[i] ? '$' : '₩'} />
-                        <br />
-                        <Button variant="link" style={{padding: '0'}} id={i} onClick={this.currencyConvert}>{this.state.currency[i] ? 'Convert To KRW' : 'Convert To USD'}</Button>
-                    </Card.Text>
-                    <Button variant={
-                        this.selected[i] ? 'secondary' : 'primary'
-                    } id={i} onClick={this.itemClicked}>{
-                        this.selected[i] ? 'Remove from cart' : 'Add to cart'
-                    }</Button>
-                </Card.Body>
-                </Card>
-            </Col>
+                    <Card style={{ marginBottom: '10px' }}>
+                        <Card.Img variant="top" src={this.state.items[i].imgUrl} style={{width: '100%', height: '20vw', objectFit: 'cover'}} />
+                        <Card.Body>
+                            <Card.Title>{this.state.items[i].name}</Card.Title>
+                            <Card.Text>
+                                <NumberFormat value={this.state.items[i].price * (this.state.currency[i] ? this.rate : 1)} displayType={'text'} thousandSeparator={true} prefix={this.state.currency[i] ? '$' : '₩'} />
+                                <br />
+                                <Button variant="link" style={{padding: '0'}} id={i} onClick={this.currencyConvert}>{this.state.currency[i] ? 'Convert To KRW' : 'Convert To USD'}</Button>
+                            </Card.Text>
+
+                            {/*<ButtonGroup>*/}
+                            {/*    <Button  id={i} onClick={this.itemMinus} disabled={!this.selected[i]} variant='secondary'>-</Button>*/}
+                            {/*    <Button  id={i} onClick={this.itemPlus} variant='primary'>+</Button>*/}
+                            {/*</ButtonGroup>*/}
+                            <InputGroup>
+                                <InputGroup.Append>
+                                    <Button id={i} onClick={this.itemMinus} disabled={!this.selected[i]} variant='secondary'>-</Button>
+                                </InputGroup.Append>
+                                <FormControl id={i}
+                                    placeholder={this.selected[i]+"개"}
```

Comment:
Why did you use `placeholder` to display the information? And why did you let the input box changeable by users?